contents/blog/posthog-vs-statsig.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: "In-depth: PostHog vs Statsig"
+date: 2023-12-06
+author: ["lior-neu-ner"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: General
+tags:
+    - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonHeader } from 'components/ComparisonTable/header'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+PostHog and Statsig both offer A/B testing and feature flags, but they're different in two important ways:
+
+- **Statsig** offers advance experimentation tools, such as multi-armed bandit and holdout testing. It's ideal for data scientists, product managers, and growth teams who need to conduct complex A/B testing.
```

Comment:
Statsig being warehouse native is a big different

Reply:
Mentioned in the table below, but I dont call this out here since I think most people are looking primarily at a/b testing comparision (it's their main feature)